ext/AbstractFFTsChainRulesCoreExt.jl

```diff
@@ -33,10 +33,12 @@ function ChainRulesCore.rrule(::typeof(rfft), x::AbstractArray{<:Real}, dims)
     project_x = ChainRulesCore.ProjectTo(x)
     function rfft_pullback(ȳ)
         dY = ChainRulesCore.unthunk(ȳ)
-        # apply scaling
+        # apply scaling; below approach is for GPU CuArray compatibility, see PR #96
         dY_scaled = similar(dY)
         dY_scaled .= dY
         dY_scaled ./= 2
```

Comment:
you could actually fuse this line with the previous one I think, e.g. `dY_scaled .= dY ./ 2` and similarly for the other scalings